doc/source/tutorial/stats.rst

```diff
@@ -1190,3 +1191,246 @@ because the p-value is very low and the MGC test statistic is relatively high.
 The MGC-map indicates a **strongly nonlinear relationship**. The optimal scale
 in this case is **equivalent to the local scale**, marked by a red spot on the
 map.
+
+Quasi-Monte Carlo
+-----------------
+
+Before talking about Quasi-Monte Carlo (QMC), a quick introduction about Monte
+Carlo (MC). The MC method was called in statistics 'model sampling'. It is
+a method which aims at trying out hypothesis. Once all the hypothesis we
+have defined have been tested, we can compute some statistics such as the
+mean of a given quantity of interest. A scenarios is defined by some variables.
+So, in order to map all scenarios, we use a random number generator.
+
+Generating random numbers is a more complex problem than it sounds. Simple
+Monte Carlo (MC) methods are designed to sample points to be independent and
+identically distributed (IID). But generating multiple sets of random points
+can produce radically different results.
+
+.. plot:: tutorial/stats/plots/qmc_plot_mc.py
+   :align: center
+   :include-source: 0
+
+In both cases, any points is taken randomly without any knowledge about the
+previous draws. And it is clear that some regions of the space are left
+unexplored. But as the number of sample increases, the space will fill.
+
+A great benefit of MC is that it has some convergence properties.
+Let's look at the mean of the square of sum in 5 dimensions:
+
+.. math::
+
+    f(\mathbf{x}) = \left( \sum_{j=1}^{5}x_j \right)^2,
+
+which has a known mean :math:`\mu = 5/3+5(5-1)/4`. Using MC sampling, we can
+compute the mean and the error follows a theoretical rate
+of :math:`O(n^{-1/2})`.
+
+.. plot:: tutorial/stats/plots/qmc_plot_conv_mc.py
+   :align: center
+   :include-source: 0
+
+Although the convergence is ensured, practitioner tend to want to have an
+exploration process which is more deterministic.
```

Comment:
Deterministic here means repeatable/knows-sequence. Yes using a seed can fix this, the default QMC sequences are fixed. So fully deterministic without seed.

Reply:
Hmm, was there a discussion about that in review? That's very counterintuitive with the default `seed=None`, and doesn't seem healthy. Perhaps I'm missing something about QMC, but for normal PRNGs this is definitely bad.

Reply:
I am not sure about what you want to say here. 

I added the following:

> Although the convergence is ensured, practitioners tend to want to have an
> exploration process which is more deterministic. With normal MC, a seed can be
> used to have a repeatable process. But fixing the seed would break the
> convergence property: a given seed could work for a given class of problem
> and break for another one.

Reply:
LGTM